### ISSUE TITLE
Two test failures with matplotlib 2.0.0b4

### ISSUE THREAD

**Issue by rathann** [mwaskom/seaborn#1025]
On Fedora rawhide/x86_64 with matplotlib 2.0.0b4, I get these two test failures:

```
======================================================================
FAIL: seaborn.tests.test_axisgrid.TestFacetGrid.test_subplot_kws
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/builddir/build/BUILDROOT/python-seaborn-0.7.1-2.fc26.x86_64/usr/lib/python2.7/site-packages/seaborn/tests/test_axisgrid.py", line 272, in test_subplot_kws
    nt.assert_equal(ax.get_axis_bgcolor(), "blue")
AssertionError: (0.0, 0.0, 1.0, 1.0) != 'blue'

----------------------------------------------------------------------
```

There are some warnings while running the above test:

```
seaborn.tests.test_axisgrid.TestFacetGrid.test_subplot_kws ... /usr/lib64/python2.7/site-packages/matplotlib/cbook.py:137: MatplotlibDeprecationWarning: The axisbg attribute was deprecated in version 2.0. Use facecolor instead.
  warnings.warn(message, mplDeprecation, stacklevel=1)
/builddir/build/BUILDROOT/python-seaborn-0.7.1-2.fc26.x86_64/usr/lib/python2.7/site-packages/seaborn/tests/test_axisgrid.py:272: MatplotlibDeprecationWarning: The get_axis_bgcolor function was deprecated in version 2.0. Use get_facecolor instead.
  nt.assert_equal(ax.get_axis_bgcolor(), "blue")
```

According to http://matplotlib.org/2.0.0b4/api/api_changes.html these are indeed deprecated. The new colors API is described here: http://matplotlib.org/2.0.0b4/api/colors_api.html .

```
======================================================================
FAIL: seaborn.tests.test_palettes.TestColorPalettes.test_mpl_reversal
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/builddir/build/BUILDROOT/python-seaborn-0.7.1-2.fc26.x86_64/usr/lib/python2.7/site-packages/seaborn/tests/test_palettes.py", line 166, in test_mpl_reversal
    nt.assert_equal(pal_forward, pal_reverse[::-1])
AssertionError: [(0.86168396770472899, 0.91280276816608996, 0.94975778546712808), (0.71146482122260668, 0.80127643214148403, 0.88830449826989621), (0.58998846597462518, 0.67472510572856592, 0.8219915417147251), (0.5490196078431373, 0.49036524413687044, 0.72867358708189156), (0.53788542868127642, 0.30269896193771628, 0.63844675124951944), (0.50943483275663204, 0.084198385236447515, 0.50302191464821222)] != [(0.86168396770472899, 0.91280276816608996, 0.94975778546712808), (0.71146482122260668, 0.80127643214148403, 0.88830449826989621), (0.58998846597462518, 0.67472510572856592, 0.8219915417147251), (0.5490196078431373, 0.49036524413687044, 0.72867358708189156), (0.53788542868127642, 0.30269896193771628, 0.63844675124951944), (0.50943483275663204, 0.084198385236447529, 0.50302191464821222)]

----------------------------------------------------------------------
```

The second one looks dubious, so I opened an issue with matplotlib upstream as well (https://github.com/matplotlib/matplotlib/issues/7193).

**Comment by rathann** [comment]
Minimal patch fixing the first failure for me (not sure if this is the correct fix):
```
diff -up seaborn-0.7.1/seaborn/tests/test_axisgrid.py.m2 seaborn-0.7.1/seaborn/tests/test_axisgrid.py
--- seaborn-0.7.1/seaborn/tests/test_axisgrid.py.m2     2016-06-05 02:34:31.000000000 +0200
+++ seaborn-0.7.1/seaborn/tests/test_axisgrid.py        2016-09-28 12:58:27.180824134 +0200
@@ -269,7 +269,7 @@ class TestFacetGrid(PlotTestCase):
 
         g = ag.FacetGrid(self.df, subplot_kws=dict(axisbg="blue"))
         for ax in g.axes.flat:
-            nt.assert_equal(ax.get_axis_bgcolor(), "blue")
+            nt.assert_equal(ax.get_axis_bgcolor(), mpl.colors.to_rgba('b'))
 
     @skipif(old_matplotlib)
     def test_gridspec_kws(self):
```

**Comment by stonebig** [comment]
please   that `get_axis_bgcolor()` has a deprecation warning, so seaborn may also do a patch, like https://github.com/bokeh/bokeh/pull/5181/files n to replace  `get_axis_bgcolor()`  per  `get_facecolor()`

**Comment by mwaskom** [comment]
That's just testing an arbitrary subplot_kw so it would be more straightforward to just change the test to a parameter that is the same in the new matplotlib.

**Comment by stonebig** [comment]
`get_axis_bgcolor` only appears in this test of seaborn project. I'll propose a patch

**Comment by mwaskom** [comment]
Closed with #1100
